### ISSUE TITLE
--release should keep the [Unreleased] section

### ISSUE THREAD

**Issue by Glandos** [oscarotero/keep-a-changelog#31]
As recommended in https://keepachangelog.com/en/1.0.0/#effort the `[Unreleased]` section should be kept at the top even if empty.

Currently, when calling `changelog --release 2.0.0`, the `[Unreleased]` is removed.

**Comment by oscarotero** [comment]
According with the spec:
> At release time, you can move the Unreleased section changes into a new release version section. 

And this is what this library does: transform the Unreleased section to the latest released version.
Do you mean creating a new empty Unreleased section?

**Comment by Glandos** [comment]
> Do you mean creating a new empty Unreleased section?

Yes, this is what I'm expecting from reading the "Effort" section. However, since the next version is… well… unknown, it could only be `[Unreleased]`, and not `[X.Y.Z] - Unreleased`.

**Comment by oscarotero** [comment]
Maybe a new option `--create`?

```sh
changelog --create  # Create a new [Unreleased] version
changelog --create 2.0.0  # Create a new [2.0.0] - Unreleased version
```

**Comment by Glandos** [comment]
This would completely fit my use case. And since it's not linked to `--release`, it's easy for other use cases, such as creating an empty section. Good idea!

**Comment by oscarotero** [comment]
Released in 2.3.0
